tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```diff
@@ -272,7 +272,9 @@ spec:
                 esac
             done
             echo
-            jq -cr '{ "state": .state, "state_reason": .state_reason }' "$(results.jsonBuildInfo.path)" | jq -Rc \
+            # buildState is a json string that should be escaped to garantee that there is no evaluation of the content
```

Comment:
s/garantee/guarantee/

Reply:
I removed the comment as it does not make sense anymore after removing the `jq -Rc`